### DISCUSSION TITLE
The Health UI displayes incorrectly When Head and Head attachments have different sourcerect and origin

### DISCUSSION THREAD

**Discussion by ToumanLin** [FakeFishGames/Barotrauma#14646] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

The incorrect health UI
![image](https://github.com/user-attachments/assets/5a21b031-193c-4b0c-8dad-0a7dd003fe98)
![image](https://github.com/user-attachments/assets/2fb3a93b-ea37-40ee-b241-d7ddfe367335)
However, the character displays correctly:
![image](https://github.com/user-attachments/assets/28884922-ba1d-4d01-ab44-1ce680b8f884)
![image](https://github.com/user-attachments/assets/1a8b8b20-0ea1-4175-8b15-b0d8fc2120d5)

### Reproduction steps

Biff_W reported issue 12068 that headattachments' sourcerects are unchangeable last year and it had been fixed, here's the link: 
https://github.com/FakeFishGames/Barotrauma/issues/12068

However, the fix only covered character limb, not about the health interface.

Reproduction steps (Make hair and head have different sourcerects):
In human.xml:
```
      <Wearable type="Hair" commonness="0.05" >
        <sprite name="Hair 1" texture="%ModDir:2809175631%/Content/Characters/Human/Headattachments/Human_decor.png" sourcerect="0,0,240,480" origin="0.6125,0.3229" inheritorigin="false" inheritsourcerect="false" />
      </Wearable>
```
In HumanDefaultRagdoll.xml
```
<limb Name="Head (1)" ID="1" Type="Head" >
    <sprite texture="%ModDir:2809175631%/Content/Characters/Human/Human_[GENDER]_heads.png" sourcerect="0,0,160,228" depth="0.05" origin="0.6125,0.33" color="255,255,255,255" deadcolor="255,255,255,255" deadcolortime="0" ignoretint="False" />
    <tintmask texture="%ModDir:2809175631%/Content/Characters/Human/Human_[GENDER]_heads_mask.png" />
</limb>
```

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by lIllIIlI** [comment]
I swear I saw Regalis post that it was fixed
![image](https://github.com/user-attachments/assets/37b87bf5-ce79-4e74-963e-568bdad7bca1)

But in current release it's not?
![Barotrauma_rnQKVZd0K2](https://github.com/user-attachments/assets/365a4a85-0a05-42d2-9314-ca67a40686b1)

**Reply by ToumanLin** [reply]
> I swear I saw Regalis post that it was fixed ![image](https://private-user-images.githubusercontent.com/131556758/379056199-37b87bf5-ce79-4e74-963e-568bdad7bca1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk2NDYwNTIsIm5iZiI6MTcyOTY0NTc1MiwicGF0aCI6Ii8xMzE1NTY3NTgvMzc5MDU2MTk5LTM3Yjg3YmY1LWNlNzktNGU3NC05NjNlLTU2OGJkYWQ3YmNhMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDIzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAyM1QwMTA5MTJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iMDNkMGJlMDYyNWQzZDA1NmY3NjFiYmY5ZDhiOTBjYTRjNDMxNjNhMGZiOWJiZmYxOWRiMTVmODBlZjIzZmRiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.k8B3ZQTAJX7VrsH2AhJ-V3twzZjw9elgsP_c9yB1Hvc)
> 
> But in current release it's not? ![Barotrauma_rnQKVZd0K2](https://private-user-images.githubusercontent.com/131556758/379056262-365a4a85-0a05-42d2-9314-ca67a40686b1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk2NDYwNTIsIm5iZiI6MTcyOTY0NTc1MiwicGF0aCI6Ii8xMzE1NTY3NTgvMzc5MDU2MjYyLTM2NWE0YTg1LTBhMDUtNDJkMi05MzE0LWNhNjdhNDA2ODZiMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDIzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAyM1QwMTA5MTJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00MWU2NTM3OWFhNWNmMzIyODAxZTUxMjlkNzRlNzNiMmZlMjI5M2FhYmZlNGE5YjE1NjUxYmU0NDE5M2MzMDljJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.lPQZw1I_iWzrMqYtwGzFdS3u24YiHmXCnrCXgGeHQVQ)

I also test myself. No, it hasn't been fixed

**Comment by Regalis11** [comment]
The fix has been now merged for the upcoming release, closing.
